### PR TITLE
Move sscs bulk scan to prod

### PR DESCRIPTION
**Pull request by jackmaloney**

### CODE REVIEW

File: k8s/prod/common/sscs/sscs-bulk-scan.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: sscs-bulk-scan
+    version: 0.0.4
+  values:
+    java:
+      replicas: 2
+      memoryRequests: '512Mi'
```
Comment (timja) [inline k8s/prod/common/sscs/sscs-bulk-scan.yaml]: not needed 21-24

File: k8s/prod/common/sscs/sscs-bulk-scan.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      cpuRequests: '250m'
+      memoryLimits: '2048Mi'
+      cpuLimits: '1500m'
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "d6c3dd58-af16-425d-8c4a-f491e895e35d"
+      image: hmctspublic.azurecr.io/sscs/sscs-bulk-scan:latest
+      environment:
+        IDAM_URL: https://idam-api.platform.hmcts.net
+        IDAM_OAUTH2_REDIRECT_URL: https://sscs-case-loader-prod.service.core-compute-prod.internal
+        JAVA_TOOL_OPTIONS: -XX:InitialRAMPercentage=30.0 -XX:MaxRAMPercentage=65.0
```
Comment (timja) [inline k8s/prod/common/sscs/sscs-bulk-scan.yaml]: not needed

File: k8s/prod/common/sscs/sscs-bulk-scan.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    java:
+      replicas: 2
+      memoryRequests: '512Mi'
+      cpuRequests: '250m'
+      memoryLimits: '2048Mi'
+      cpuLimits: '1500m'
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "d6c3dd58-af16-425d-8c4a-f491e895e35d"
+      image: hmctspublic.azurecr.io/sscs/sscs-bulk-scan:latest
+      environment:
```
Comment (timja) [inline k8s/prod/common/sscs/sscs-bulk-scan.yaml]: i noticed this in your default config:
https://github.com/hmcts/sscs-bulk-scan/blob/master/charts/sscs-bulk-scan/values.yaml#L14-L17
They are very old tactical values which unless you're relying on them should probably be deleted?

Comment (jackmaloney) [reply k8s/prod/common/sscs/sscs-bulk-scan.yaml]: How can I tell if we are relying on them? Remove and see what happens?

Comment (timja) [reply k8s/prod/common/sscs/sscs-bulk-scan.yaml]: probably, it's unlikely you are

File: k8s/prod/common/sscs/sscs-bulk-scan.yaml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "d6c3dd58-af16-425d-8c4a-f491e895e35d"
+      image: hmctspublic.azurecr.io/sscs/sscs-bulk-scan:latest
+      environment:
+        IDAM_URL: https://idam-api.platform.hmcts.net
+        IDAM_OAUTH2_REDIRECT_URL: https://sscs-case-loader-prod.service.core-compute-prod.internal
+      aadIdentityName: sscs
```
Comment (timja) [inline k8s/prod/common/sscs/sscs-bulk-scan.yaml]: this shouldn't be here, it's coming from values.yaml here:
https://github.com/hmcts/sscs-bulk-scan/blob/master/charts/sscs-bulk-scan/values.yaml#L4

Comment (timja) [reply k8s/prod/common/sscs/sscs-bulk-scan.yaml]:
```suggestion
```

File: k8s/prod/common/sscs/sscs-bulk-scan.yaml [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    global:
+      environment: prod
+      subscriptionId: "8999dec3-0104-4a27-94ee-6588559729d1"
```
Comment (timja) [inline k8s/prod/common/sscs/sscs-bulk-scan.yaml]:
```suggestion
```

File: k8s/prod/common/sscs/sscs-tribunals-api.yaml [hunk 6]
```diff
@@ -30,11 +26,7 @@ spec:
         IDAM_URL: https://idam-api.platform.hmcts.net
         IDAM_OAUTH2_REDIRECT_URL: https://sscs-case-loader-prod.service.core-compute-prod.internal
         DOC_ASSEMBLY_URL: "http://dg-docassembly-prod.service.core-compute-prod.internal"
-        JAVA_TOOL_OPTIONS: -XX:InitialRAMPercentage=30.0 -XX:MaxRAMPercentage=65.0
```
Comment (timja) [inline k8s/prod/common/sscs/sscs-tribunals-api.yaml]: this still needs to be here on tribunals until you upgrade your docker image version

File: k8s/prod/common/sscs/sscs-evidence-share.yaml [hunk 7]
```diff
@@ -33,12 +26,7 @@ spec:
         PDF_SERVICE_BASE_URL: https://docmosis.platform.hmcts.net/rs/render
         PDF_SERVICE_HEALTH_URL: https://docmosis.platform.hmcts.net/rs/status
         IDAM_OAUTH2_REDIRECT_URL: https://sscs-case-loader-prod.service.core-compute-prod.internal
-        JAVA_TOOL_OPTIONS: -XX:InitialRAMPercentage=30.0 -XX:MaxRAMPercentage=65.0
```
Comment (timja) [inline k8s/prod/common/sscs/sscs-evidence-share.yaml]: this still needs to be here until you update your base docker image version:
https://github.com/hmcts/sscs-evidence-share/blob/master/Dockerfile#L3

File: k8s/prod/common/sscs/sscs-bulk-scan.yaml [hunk 8]
```diff
@@ -0,0 +1,30 @@
+---
```
Comment (timja) [inline k8s/prod/common/sscs/sscs-bulk-scan.yaml]: needs secops approval before we can move this, will try get it today

**Pull request merged**